package.json

```diff
@@ -36,10 +38,12 @@
     "clean:lerna": "lerna clean --yes --parallel --loglevel=silent",
     "build": "lerna run build --parallel --loglevel=silent",
     "build:current": "lerna run build:current --parallel --loglevel=silent",
-    "build:full": "npm run clean:lerna && npm run bootstrap && npm run build && npm run mocha && npm run lint",
+    "build:full":
+      "npm run clean:lerna && npm run bootstrap && npm run build && npm run mocha && npm run lint",
     "pretest": "npm run clean && npm run build:current",
     "test": "node packages/build/bin/run-nyc npm run mocha",
-    "mocha": "node packages/build/bin/run-mocha \"packages/*/DIST/test/**/*.js\" \"packages/cli/test/*.js\"",
+    "mocha":
+      "node packages/build/bin/run-mocha \"packages/*/DIST/test/**/*.js\" \"packages/cli/test/*.js\"",
```

Comment:
The formatting changes for `package.json` should be reverted. Make sure you IDE won't format it automatically.